src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/AssemblyReference.xaml

```diff
@@ -27,6 +27,24 @@
                   DisplayName="Embed Interop Types"
                   Description="Indicates whether types defined in this assembly will be embedded into the target assembly." />
 
+    <StringProperty Name="Identity"
+                    ReadOnly="True"
+                    DisplayName="Identity"
+                    Description="Security identity of the referenced assembly (see System.Reflection.Assembly.Evidence or System.Security.Policy.Evidence).">
+        <StringProperty.DataSource>
+            <DataSource PersistedName="{}{Identity}" SourceOfDefaultValue="AfterContext" />
+        </StringProperty.DataSource>
+    </StringProperty>
+
+    <StringProperty Name="ResolvedPath"
+                    ReadOnly="True"
+                    DisplayName="Path"
+                    Description="Location of the file being referenced.">
+        <StringProperty.DataSource>
+            <DataSource PersistedName="Identity" SourceOfDefaultValue="AfterContext" />
```

Comment:
@davkean Honestly, I don't know. This is copied as-is from ResolvedAssemblyReference.xaml.

Now, my *guess* is that it isn't really relevant. Since this is a property for an unresolved assembly reference I wouldn't expect us to have a meaningful path; otherwise, it wouldn't be unresolved. Do you think I could/should remove the `StringProperty.DataSource` element entirely?